### PR TITLE
[Test] Add tests to check for empty steps and jumbled order of tags

### PR DESCRIPTION
**Pull request by kaijie0102**
addRecipe_emptySteps_exceptionThrown()

addValidCommand_stepAndIngredientNotInOrder_recipeCreated()

### CODE REVIEW

File: src/test/java/essenmakanan/parser/AddRecipeCommandTest.java [hunk 1]
```diff
@@ -103,12 +103,36 @@ public void addRecipeWithInvalidInput_invalidIngredient_errorThrown() {
...
+        addRecipeCommand = new AddRecipeCommand(userInput, recipeList);
+        addRecipeCommand.executeCommand();
+        assertEquals("bread", recipeList.getRecipe(0).getTitle());
+        String step1 = recipeList.getRecipe(0).getRecipeSteps().getStepByIndex(0).getDescription();
+        String step2 = recipeList.getRecipe(0).getRecipeSteps().getStepByIndex(1).getDescription();
+        assertEquals("step 1 instructions", step1);
+        assertEquals("step 2 instructions", step2);
+    }
+
+
```
Comment (StanleyW00) [inline src/test/java/essenmakanan/parser/AddRecipeCommandTest.java:137]: Nice coverage of tests

**Review by StanleyW00 — APPROVED** [review]
Overall, LGTM.
